### PR TITLE
MDCT-410: add section a - add plans

### PR DESCRIPTION
**Pull request by gmrabian**
## Description
[MDCT-410](https://qmacbis.atlassian.net/browse/MDCT-410) Add the "Add Plans" section of Section A. 

Honestly I'm floored this is all it took.

### How to test
- `./dev local`
- Navigate to `/mcpar/program-information/add-plans`
- test it out, fill it out, etc.

If you want you can run dynamodb admin to check it's being updated

## Code author checklist
- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by braxex — APPROVED** [review]
It's beautiful. Long live the form factory.

**Review by karla-vm — APPROVED** [review]
✨

File: services/ui-src/src/forms/mcpar/aap/aap.json [hunk 1]
```diff
@@ -2,17 +2,20 @@
   "path": "/mcpar/program-information/add-plans",
   "intro": {
     "section": "Section A: Program Information",
-    "subsection": "Add Plans"
+    "subsection": "Add plans (A.7)",
```
Comment (ntsummers1) [inline services/ui-src/src/forms/mcpar/aap/aap.json:5]: If I'm not mistaken the P should be capitalized here

File: services/ui-src/src/forms/mcpar/aap/aap.json [hunk 2]
```diff
@@ -2,17 +2,20 @@
...
+    },
     "fields": [
       {
         "id": "aap-1",
-        "type": "text",
+        "type": "dynamic",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "Plan name"
```
Comment (ntsummers1) [inline services/ui-src/src/forms/mcpar/aap/aap.json:18]: N should be capitalized here too

**Comment by braxex** [comment]
I think our convention so far has been to only have first word capitalized, but these dynamic fields are admittedly kind of a different case. Worth checking with design.

**Comment by gmrabian** [comment]
Checking with David on these. The discrepancy is between Jira and Figma

File: services/ui-src/src/forms/mcpar/aap/aap.json (hunk 1 above)
Comment (gmrabian) [reply services/ui-src/src/forms/mcpar/aap/aap.json:5]: Got confirmation from David to use lowercase, like we've been doing for other pages

File: services/ui-src/src/forms/mcpar/aap/aap.json (hunk 2 above)
Comment (gmrabian) [reply services/ui-src/src/forms/mcpar/aap/aap.json:18]: Got confirmation from David to use lowercase, like we've been doing for other pages

**Comment by gmrabian** [comment]
Got confirmation from David to use lowercase, like we've been doing for other pages

**Pull request merged**